packages/kununu-header/Header.jsx

```diff
@@ -11,25 +11,53 @@ export default function Header ({
   assetsPath,
   title,
   logoLink,
+  simple,
 }) {
+  const simpleLogo = () => {
+    const content = (
+      <picture>
+        <source
+          srcSet={`${assetsPath}/logo-desktop.svg`}
+        />
+        <img
+          alt={title}
+          aria-hidden="true"
+          className={styles.simpleLogo}
+          rel="presentation"
+          src={`${assetsPath}/logo-desktop.svg`}
+        />
+      </picture>
+    );
+
+    return logoLink ?
+      React.cloneElement(logoLink, logoLink.props, content) :
+      content;
+  };
+
   return (
     <header
       role="banner"
-      className={`${styles.header} ${fixed && styles.fixed}`}
+      className={`${styles.header} ${fixed && styles.fixed} ${simple && styles.simple}`}
     >
       <div className={container}>
         <div className={styles.flex}>
           <div className={styles.pullLeft}>
-            <Logo
-              link={logoLink}
-              assetsPath={assetsPath}
-              title={title}
-            />
+            {simple ? simpleLogo() : (
```

Comment:
I have a feeling, that this whole logic is a little bit overcomplicated written. Maybe it makes sense to abstract out the component for Simple Logo and Logo to it's own component? Also will the Simple Logo ever not have a logo link? React.cloneElement always seems a little bit heavy to me, so maybe this can be refactored as well.

Reply:
So you mean passing the "simple" prop on to the logo component and doing this comparison there?

The header logo will always be a link as far as I know. If we don't clone we would need to change all the apps to pass in a url and always have an anchor within the component. This would mean breaking changes for both header and logo. Or I could try grabbing the attribute off of the element passed in, but I'm not sure that is much nicer 🤔

Reply:
Same as above comment